main.py

```diff
@@ -1,7 +1,43 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        ["George", "Klooney"],
+        ["Kianu", "Reaves"],
+        ["Scarlett", "Keegan"],
+        ["Will", "Smith"],
+        ["Jaden", "Smith"],
+        ["Scarlett", "Johansson"]
+    ]
+
+    for actor in actors:
```

Comment:
You can unpack lists in place, if you want. For example,
```python
for value1, value2 in list_of_pairs:

Reply:
You can sure use a dictionary if you want to, but I don't think that's necessary. I don't even think that would be an improvement.

Reply:
Ok, I'll leave it like this, then

Reply:
you don't need a dictionary to unpack tuples in the cycle)